src/model/core/discharge/investment_discharge.jl

```diff
@@ -165,14 +165,28 @@ function investment_discharge!(EP::Model, inputs::Dict, setup::Dict)
 	# DEV NOTE: This constraint may be violated in some cases where Existing_Cap_MW is <= Min_Cap_MW and lead to infeasabilty
 	@constraint(EP, cMinCap[y in intersect(dfGen[dfGen.Min_Cap_MW.>0, :R_ID], 1:G)], eTotalCap[y] >= dfGen[y, :Min_Cap_MW])
 
+
+	x = inputs["NumberOfMinCapReqs"]
```

Comment:
This line potentially causes an error if NumberOfMinCapReqs is not defined. At the least, please put this inside of the `if` statement. And please don't use single-letter variable names - prefer descriptive ones like `num_mincapreqs`